Homework/Program.cs

```diff
@@ -4,7 +4,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+            // Prvy objekt car1
+            var car1 = new Car("Skoda", "Felicia", 1999, 300000);
+            car1.DisplayCarInfo();
+            Console.WriteLine("Počet kilometrov zvýšený o 123: " + car1.Drive(123) + " km");
+            Console.WriteLine("");
+
+            // Druhy objekt car2
+            var car2 = new Car("Volkswagen", "Golf", 2004, 170000);
+            car2.DisplayCarInfo();
+            Console.WriteLine("Počet kilometrov zvýšený o 55000: " + car2.Drive(55000) + " km");
+
+            Console.WriteLine("");
```

Comment:
Pro zajimavost: Write metoda je definovana i bez parametru. Pak se chova stejne jako to, ceho jsi tady docilil pomoci prazdneho retezce
```suggestion
            Console.WriteLine();
```